tests/client/amo/components/TestAddonDetail.js

```diff
@@ -10,7 +10,7 @@ import AddonDetail, { allowedDescriptionTags }
 import I18nProvider from 'core/i18n/Provider';
 import InstallButton from 'core/components/InstallButton';
 
-import { getFakeI18nInst } from 'tests/client/helpers';
+import { getFakeI18nInst } from '../../helpers';
```

Comment:
Is there a way to not need non-relative imports here. This gets really messy fast. Also it makes test code less portable.